frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.styled.tsx

```diff
@@ -10,12 +10,14 @@ import { SIDEBAR_WIDTH } from "../Sidebar";
 export const DashboardInfoSidebarRoot = styled.aside`
   width: ${SIDEBAR_WIDTH}px;
   min-width: ${SIDEBAR_WIDTH}px;
-  padding: 0 2rem 0.5rem;
   background: ${color("white")};
   border-left: 1px solid ${color("border")};
   align-self: stretch;
-  overflow-y: auto;
+  // FIXME: ensure that removing this style is OK
+  // overflow-y: auto;
```

Comment:
not ok 🥴 , we want the sidebar to scroll internally

![scrollyuck](https://github.com/metabase/metabase/assets/30528226/bc7c981f-5754-48e2-a8bb-b6b51ad626cc)

Reply:
The parent sidebar doesn't scroll internally. I think adding this functionality might work well in a follow-up PR. Wdyt?

![Kapture 2024-05-14 at 07 34 07](https://github.com/metabase/metabase/assets/130925/0620806e-1057-4797-b7eb-83229f5ee2f5)